selftests/functional/plugin/spawners/podman.py

```diff
@@ -116,3 +117,19 @@ def test_asset_files(self):
         self.assertEqual(result.exit_status, 0)
         self.assertIn("use_data.sh: STARTED", result.stdout_text)
         self.assertIn("use_data.sh:  PASS", result.stdout_text)
+
+
+class OperationalTest(Test):
+    def test_not_operational(self):
+        fake_podman_bin = os.path.join(BASEDIR, "examples", "tests", "false")
+        result = process.run(
+            f"{AVOCADO} run "
+            f"--job-results-dir {self.workdir} "
+            f"--disable-sysinfo --spawner=podman "
+            f"--spawner-podman-bin={fake_podman_bin} "
+            f"--spawner-podman-image=fedora:36 -- "
```

Comment:
I would remove the podman-image from this test. It won't change the behavior of the test, and we wouldn't have to deal with updating the test based on fedora support in the future.